src/AppShed/Extensions/SpreadsheetBundle/Controller/ReadController.php

```diff
@@ -160,6 +184,36 @@ public function documentAction(Request $request)
         }
     }
 
+    function readGeocode($address){
```

Comment:
This function needs to be rewritten using Guzzle.

This is mostly so that it has some proper error handling